src/mono/wasm/Wasm.Build.Tests/Blazor/WorkloadRequiredTests.cs

```diff
@@ -141,10 +141,11 @@ await BlazorRunTest(new BlazorRunOptions()
         }
         else
         {
+            string nativeName = IsWorkloadWithMultiThreadingForDefaultFramework ? "es-ES" : "espa\u00F1ol (Espa\u00F1a)"; // MT does not use JS to get the full name
```

Comment:
The idea is to make this test pass locally when running "all" WBT tests and have MT runtime build, am I correct?

Reply:
Thanks, missed this. It should be changed same as InvariantTests, it's currently run with `[Theory, TestCategory("no-workload")]1, so it always gives the long `NativeName` but the value in fact depends on the `WasmEnabledThreads`, not on presence of workload.

Reply:
I had a misconception about WBT, 2 versions of output are not necessary - we should expect the value that the app is build with, in this case, ST.